src/data/markdown/docs/03 cloud/01 Creating and running a test/02 Cloud tests from the CLI/200 Cloud execution reference.md

```diff
@@ -41,6 +41,8 @@ export const options = {
 | staticIPs (boolean)           | `false` by default                                                                     | When set to `true` the cloud system will use dedicated IPs assigned to your organization to execute the test.                                                                                                         |
 | note (string)                 | Empty by default.                                                                      | Notes regarding the test, changes made, or anything that may be worth noting about your test.                                                                                                                         |
 | deleteSensitiveData (boolean) | False by default                                                                       | If set to `true`, k6 deletes sensitive data as soon as the test starts running or, if still queued, when the test aborts. Sensitive data includes scripts, HAR files, archives, and APM credentials.                  |
+| drop_metrics (array)          | Empty by default                                                                       | Drops listed metrics during ingestion time. E.g. `["http_req_tls_handshaking"]`. This helps reduce the cardinality of time series.                                                                                    |
+| drop_tags (object)            | Empty by default                                                                       | Drops listed tags during ingestion time. E.g. `{"http_req_duration": ["instance_id"]}`. This helps reduce the cardinality of time series.                                                                             |
```

Comment:
>Drops listed tags during ingestion t

I probably won't review this until today, but for a quick preliminary question: what are "Listed metrics"? The metrics calculated? Or the ones stored and shown in the UI? It seems like it calculates, but doesn't store in the DB (and therefore isn't shown of course).

Reply:
Or maybe are they metrics listed in an array in the script somewhere?

Reply:
By listed I meant the ones in the array. As you didn't understand, I can propose:

```suggestion
Drops the metrics listed in the array during ingestion time.
```

Reply:
By metrics, I mean metrics from this list: https://k6.io/docs/using-k6/metrics/#http-specific-built-in-metrics